requirements.txt

```
@@ -1,7 +1,7 @@
 # Install requirements for MDTools in alphabetical order.
 
 matplotlib >=3.3, <4.0
-MDAnalysis >=1.0, <2.0
+MDAnalysis >=1.0, <3.0
```

```suggestion
MDAnalysis >=2.0, <3.0
```
MDAnalysis version 1 and 2 are incompatible. If we allow MDAnalysis version 2, we must disallow MDAnalysis version 1